File: Modul 5/Python/PRAK505-2410817120018-ArindaDitaPratiwi.py
def Biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir
    print(f"Perkenalkan Nama Saya: {nama}")
    print(f"Umur Saya: {umur}")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {asal}")
    print(" ")

tahun_lahir = int(input())
nama = input().strip()
asal = input().strip()
print(" ")
Biodata(tahun_lahir, nama, asal)

File: Modul 5/C/PRAK505-2410817120018-ArindaDitaPratiwi.c
#include <stdio.h>
void Biodata(int tahunLahir, char nama[], char asal[]){ 
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahunLahir;
    printf("Perkenalkan Nama Saya: %s\n", nama);
    printf("Umur Saya: %d\n", umur);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya dari: %s\n", asal);
    printf(" ");}

int main(){
    int tahunLahir;
    char nama[100], asal[50];
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", nama);
    scanf(" %[^\n]%*c", asal); 
    printf("\n");
    Biodata(tahunLahir, nama, asal);
    return 0;}